src/index.html

```diff
@@ -1,22 +1,385 @@
+                  <a
+                    href="#"
+                    class="category__link"
+                  >
+                    <img
+                      src="image/categories/earphones-first.png"
+                      alt="earphones"
+                      class="category__photo"
+                    />
+                  </a>
+                  <a
+                    href="#"
+                    class="category__link"
+                  >
+                    <img
+                      src="image/categories/earphones-second.png"
+                      alt="earphones"
+                      class="category__photo"
+                    />
+                  </a>
+                </div>
+                <h3 class="category__title">Earphones</h3>
+              </article>
+
+              <article class="category">
+                <div class="category__photos">
+                  <a
+                    href="#"
+                    class="category__link"
+                  >
+                    <img
+                      src="image/categories/smart-home-first.png"
+                      alt="smart-home"
+                      class="category__photo"
+                    />
+                  </a>
+                  <a
+                    href="#"
+                    class="category__link"
+                  >
+                    <img
+                      src="image/categories/smart-home-second.png"
+                      alt="smart-home"
+                      class="category__photo"
+                    />
+                  </a>
+                </div>
+                <h3 class="category__title">Smart Home</h3>
+              </article>
+              <button class="category__button">View all</button>
+            </div>
+          </section>
+          <div class="header-about-us"></div>
+          <section
+            class="about-us"
+            id="about-us"
+          >
+            <h2 class="section-title about-us__title">
+              Timeless, for 50 years and counting
+            </h2>
+            <p class="about-us__description">
+              Bang & Olufsen is expanding its Recreated Classics Program with
+              the launch of Beosystems – a limited edition music system that
+              transcends time by bridging the gap between one of our iconic
+              designs from 1972 and today’s cutting-edge digital technology.
+            </p>
+          </section>
+          <section
+            class="section-contact-us"
+            id="contact-us"
+          >
+            <h2 class="section-title">Contact us</h2>
+
+            <div class="contact-us">
+              <form
+                onsubmit="this.reset(); return false;"
+                class="contact-us__form"
+              >
+                <input
+                  class="contact-us__inputs contact-us__name"
+                  type="text"
+                  placeholder="Name"
+                />
+                <input
```

Comment:
When I hove this item, there is a flickering UI happening that is because you are adding a specific border valeu on hover.

To prevent that you should also add the border when the input is not hovered. You can put this border with the same color as the background of the input, in that way you will not have a border different than the background and it will also fix the flickering issue.